resources/kcp/templates/migrator-job.yaml

```diff
@@ -77,7 +77,12 @@ spec:
                     - "/bin/bash"
                   args:
                     - "-c"
+                    {{if eq .Values.global.database.embedded.enabled false}}
+                    # There is no Istio sidecar injected so there is no need to quit it
+                    - "sleep 20; ./run.sh; exit_code=$?; echo '# KILLING PILOT-AGENT #'; pkill -INT cloud_sql_proxy; sleep 5; exit $exit_code;"
+                    {{else}}
                     - "sleep 20; ./run.sh; exit_code=$?; echo '# KILLING PILOT-AGENT #'; pkill -INT cloud_sql_proxy; curl -XPOST http://127.0.0.1:15020/quitquitquit; sleep 5; exit $exit_code;"
```

Comment:
I think you should not kill clould_sql_proxy in embedded database mode. It does not exist, since it is created only in  `Values.global.database.embedded.enabled false` mode (see line 25)